### DISCUSSION TITLE
email alerts

### DISCUSSION THREAD

**Discussion by ivapeJ** [community/community#142811] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I keep getting emails from github and i cant figure out how tp get rid of them. they are all related to a git i once used but no longer need. [RSSHub]. step by sptep how do i make all the emails relating to this stop completeley

**Comment by MasterInQuestion** [comment]
Reconfig more sensible notification channels:
    https://github.com/settings/notifications
